src/cityreader.py

```diff
@@ -3,6 +3,15 @@
 
 # TODO
 
+class City:
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
+    
+    def printCustom(self):
```

Comment:
Awesome! I dig the custom method. You could also rename it to `def __str__` so in your for loop, you can just `print(c)` instead of calling `print(c.printCustom())`